flexget/plugins/output/pushover.py

```diff
@@ -101,6 +105,21 @@ def on_task_output(self, task, config):
                 log.warning("Problem rendering 'urltitle': %s" % e)
                 urltitle = ""
 
+            # Attempt to render the priority field
+            if isinstance(priority, basestring):
+                try:
+                    priority = entry.render(priority)
+                except ValueError as e:
+                    log.warning('Problem rendering "priority": %s' % e)
+                    priority = 0
```

Comment:
Priority should be sent as an int to the api. If it's not its ignored and set to 0 by default at pushover side so there's no harm.